src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
```

Comment:
I'm weakly against the type proliferation - `std::pair<RippleAddress, RippleAddress>` makes sense to me. Or even `std::pair<AnyPublicKey, AnySecretKey>`.